app/presenters/licences/licence-set-up.presenter.js

```diff
@@ -0,0 +1,139 @@
+'use strict'
+
+/**
+ * Formats data for the `/licences/{id}/licence-set-up` view licence set up page
+ * @module LicenceSetUpPresenter
+ */
+
+const { formatLongDate } = require('../base.presenter.js')
+
+const roles = {
+  billing: 'billing',
+  workflowEditor: 'charge_version_workflow_editor',
+  workflowReviewer: 'charge_version_workflow_reviewer'
+}
+
+/**
+ * Formats data for the `/licences/{id}/licence-set-up` view licence set up page
+ *
+ * @returns {Object} The data formatted for the view template
+ */
+function go (chargeVersions, workflows, auth, licence) {
+  return {
+    ..._authorisedLinks(auth, licence),
+    chargeInformation: _chargeInformation(chargeVersions, workflows, auth)
+  }
+}
+
+function _authorisedLinks (auth, licence) {
+  console.log('Date: ', new Date())
+  if (auth.credentials?.scope?.includes(roles.workflowEditor) && _isLessThanSixYearsOld(licence.startDate)) {
```

Comment:
Have you encountered a situation where `credentials` or `scope` is not defined?

My understanding is we always set these. So, (assuming I'm correct!) I think its better to believe they exist. If we encounter a situation where they don't, we'll know about it, rather than risk quietly swallowing/hiding the issue.

Reply:
This is a result of TDD where the object may not exists in the initial test build up. 

I can remove the optional channing and ensure all tests have the desired auth object inc credentials and scope.